Tests/UnitTests/EdgePublicAPITests.swift

```diff
@@ -161,6 +161,7 @@ class EdgePublicAPITests: XCTestCase {
                                                           data: [:])
             MobileCore.dispatch(event: responseEvent)
         }
+        sleep(1)
```

Comment:
can we decrease this to ms? and do we need to add this wait for the other similar tests that register a listener before dispatch?